lib/admin.php

```diff
@@ -92,87 +92,70 @@ function wc_admin_register_pages() {
 			56 // After WooCommerce & Product menu items.
 		);
 
-		wc_admin_register_page(
+		$report_pages = array(
 			array(
 				'title'  => __( 'Revenue', 'woocommerce-admin' ),
 				'parent' => '/analytics/revenue',
 				'path'   => '/analytics/revenue',
-			)
-		);
-
-		wc_admin_register_page(
+			),
 			array(
 				'title'  => __( 'Orders', 'woocommerce-admin' ),
 				'parent' => '/analytics/revenue',
 				'path'   => '/analytics/orders',
-			)
-		);
-
-		wc_admin_register_page(
+			),
 			array(
 				'title'  => __( 'Products', 'woocommerce-admin' ),
 				'parent' => '/analytics/revenue',
 				'path'   => '/analytics/products',
-			)
-		);
-
-		wc_admin_register_page(
+			),
 			array(
 				'title'  => __( 'Categories', 'woocommerce-admin' ),
 				'parent' => '/analytics/revenue',
 				'path'   => '/analytics/categories',
-			)
-		);
-
-		wc_admin_register_page(
+			),
 			array(
 				'title'  => __( 'Coupons', 'woocommerce-admin' ),
 				'parent' => '/analytics/revenue',
 				'path'   => '/analytics/coupons',
-			)
-		);
-
-		wc_admin_register_page(
+			),
 			array(
 				'title'  => __( 'Taxes', 'woocommerce-admin' ),
 				'parent' => '/analytics/revenue',
 				'path'   => '/analytics/taxes',
-			)
-		);
-
-		wc_admin_register_page(
+			),
 			array(
 				'title'  => __( 'Downloads', 'woocommerce-admin' ),
 				'parent' => '/analytics/revenue',
 				'path'   => '/analytics/downloads',
-			)
-		);
-
-		if ( 'yes' === get_option( 'woocommerce_manage_stock' ) ) {
-			wc_admin_register_page(
-				array(
-					'title'  => __( 'Stock', 'woocommerce-admin' ),
-					'parent' => '/analytics/revenue',
-					'path'   => '/analytics/stock',
-				)
-			);
-		}
-
-		wc_admin_register_page(
+			),
 			array(
 				'title'  => __( 'Customers', 'woocommerce-admin' ),
 				'parent' => '/analytics/revenue',
 				'path'   => '/analytics/customers',
-			)
-		);
-
-		wc_admin_register_page(
+			),
 			array(
 				'title'  => __( 'Settings', 'woocommerce-admin' ),
 				'parent' => '/analytics/revenue',
 				'path'   => '/analytics/settings',
-			)
+			),
 		);
+
+		if ( 'yes' === get_option( 'woocommerce_manage_stock' ) ) {
```

Comment:
Optional nitpick: maybe we could add this to the above array and check if it not null before registering the page?  If someone were to register there page at the beginning of the array or remove pages, we might lose ordering here.

```php
			'yes' === get_option( 'woocommerce_manage_stock' ) ? array(
				'title'  => __( 'Stock', 'woocommerce-admin' ),
				'parent' => '/analytics/revenue',
				'path'   => '/analytics/stock',
			) : null,
```